test/integration/isolated/examples_udproute_test.go

```diff
@@ -32,25 +33,28 @@ func TestExampleUDPRoute(t *testing.T) {
 		WithSetup("deploy kong addon into cluster", featureSetup(
 			withControllerManagerOpts(helpers.ControllerManagerOptAdditionalWatchNamespace("default")),
 		)).
-		Assess("deploying to cluster works and deployed coredns responds to UDP queries",
+		Assess("deploying to cluster works and udp traffic is routed to the service",
 			func(ctx context.Context, t *testing.T, _ *envconf.Config) context.Context {
 				cleaner := GetFromCtxForT[*clusters.Cleaner](ctx, t)
 				cluster := GetClusterFromCtx(ctx)
-				proxyUDPURL := GetUDPURLFromCtx(ctx)
+				// GetUDPURLFromCtx returns the URL of the UDP service, but with http prefix
+				// http://<IP>:<PORT> (bug in KTF), taking the Host part trims scheme part.
+				proxyUDPURL := GetUDPURLFromCtx(ctx).Host
 
 				t.Logf("applying yaml manifest %s", udpRouteExampleManifests)
 				b, err := os.ReadFile(udpRouteExampleManifests)
-				assert.NoError(t, err)
+				require.NoError(t, err)
```

Comment:
I remember that `require`s where breaking the whole test suite when they'd fail (I don't recall the specific effect of that though). I'm pretty sure the clean up didn't work.

Let's leave that in. To improve the test we can return if `err != nil`